apollo-runtime/src/main/java/com/apollostack/android/GraphqlService.java

```diff
@@ -0,0 +1,14 @@
+package com.apollostack.android;
+
+import android.database.Observable;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/** A {@link retrofit2.Retrofit} service for executing GraphQL requests. */
+public interface GraphqlService {
```

Comment:
So how API will look like from user perspective, do I need to extend/implement this interface?